### DISCUSSION TITLE
Add HP numbers/percentage on the HP BAR overhead

### DISCUSSION THREAD

**Discussion by Yanlucas01** [runelite/runelite#17116] [Ideas]
Coming from other MMO's including Runescape 3, I find the OSRS hp bar very unhelpful   
You can't tell how much HP the monster have just by looking how much of the green bar is red   
   
HP quantity inside the bar instead of top left corner would be visually cleaner and more intuitive imo   
But I can't find a plugin for it   
   

Something like this

![hpbar](https://github.com/runelite/runelite/assets/95941361/c4b555e9-da4f-4336-82c9-73890ecd8d44)

**Comment by mitchelltmarino** [comment]
Is this still something you are interested in?
